app/assets/javascripts/backbone/views/application_view.coffee

```diff
@@ -39,6 +39,11 @@ class MS.Views.ApplicationView extends Backbone.View
 
   handleAjaxErrors: ->
     $(document).ajaxError (e, jqXHR) =>
+      # Don't capture error on form validation errors.
+      # File uploads do some strange things, hence the 200 and 0
+      status = parseInt(jqXHR.status)
+      return if status == 422 || status == 0 || status == 200
```

Comment:
I could clean this up, but is there a better way than looking at the status code in the error handler?